diff --git a/node_modules/naudiodon/index.js b/node_modules/naudiodon/index.js
index 4d8537b..e886800 100644
--- a/node_modules/naudiodon/index.js
+++ b/node_modules/naudiodon/index.js
@@ -16,9 +16,9 @@
 const { Readable, Writable, Duplex } = require('stream');
 const portAudioBindings = require("bindings")("naudiodon.node");
 
-var SegfaultHandler = require('segfault-handler');
+/* var SegfaultHandler = require('segfault-handler');
 SegfaultHandler.registerHandler("crash.log");
-
+ */
 exports.SampleFormatFloat32 = 1;
 exports.SampleFormat8Bit = 8;
 exports.SampleFormat16Bit = 16;
